sonic-sensormond/tests/mock_platform.py

```diff
@@ -0,0 +1,274 @@
+from sonic_platform_base import chassis_base
+from sonic_platform_base import module_base
+from sonic_platform_base import sensor_base
+
+class MockVsensor(sensor_base.VsensorBase):
```

Comment:
These names will all be updated based on the comments in the other PRs? eg. `class MockVoltageSensor(sensor_base.VoltageSensorBase)`

Applies to all classes, strings, etc. below.

Reply:
yes